org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -176,6 +176,14 @@ Parameters:
   agoraDevDeveloperGroup:   #JC aws-agoradev-developers
     Type: String
     Default: '906769aa66-97799c36-3766-4b50-9a0a-62b22d9a5c60'
+
+  IncludeChopProdAdminGroup:     #JC aws-includechopprod-admins
+    Type: String
+    Default: 'TBD'
+
+  IncludeChopProdDeveloperGroup:   #JC aws-includechopprod-developers
+    Type: String
+    Default: 'TBD'
```

Comment:
should be `906769aa66-24060cf5-7100-4c45-a9be-27c3ff5aba69`

Reply:
Thanks!